lightning/src/ln/payment_tests.rs

```diff
@@ -499,7 +500,6 @@ fn do_retry_with_no_persist(confirm_before_reload: bool) {
 	let bs_htlc_claim_txn = nodes[1].tx_broadcaster.txn_broadcasted.lock().unwrap().split_off(0);
 	assert_eq!(bs_htlc_claim_txn.len(), 1);
 	check_spends!(bs_htlc_claim_txn[0], as_commitment_tx);
-	expect_payment_forwarded!(nodes[1], nodes[0], nodes[2], None, false, false);
```

Comment:
Ah, we want to move this because the `PaymentForwarded` event is emitted by the `handle_update_fulfill_htlc` already and then the `expect_channel_ready_event` call in `create_announced_chan_between_nodes` fails due to 2 events being queued.